integration-tests/pipelines/e2e-tests-kind-pipeline.yaml

```diff
@@ -0,0 +1,406 @@
+---
+apiVersion: tekton.dev/v1beta1
```

Comment:
Can this just be v1? Any reason to keep it v1beta1?

Reply:
Updated to v1. Thanks